package.json

```diff
@@ -29,13 +29,10 @@
     "devDependencies": {
         "@changesets/cli": "2.26.1",
         "@changesets/changelog-github": "0.4.8",
-        "@types/node": "18.15.12",
-        "@types/jest": "29.5.1",
```

Comment:
@types/jest This should break the build. jest.config.ts is in TS and uses jest.config.ts. It's ok because there is no tests at the root right now

Reply:
I doubth there will ever be any tests at the root. That's why Jest is configured with `projects`.